content/en/docs/started/workstation/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - RFMVasconcelos
+  - 8bitmp3
+
+reviewers:
+  - alfsuse
```

Comment:
Note, OWNERS inherit through the tree, you don't need to duplicate anyone here

Reply:
@8bitmp3

Reply:
Thanks @Bobgy !